src/goto-programs/goto_convert.cpp

```diff
@@ -1258,6 +1258,7 @@ void goto_convertt::convert_switch(
 
   goto_programt tmp_cases;
 
+  size_t case_number=1;
```

Comment:
Also I recommend setting this in the language frontend (perhaps renamed switch_case_description) since different languages will be using switches for different things, and it will enable us to treat e.g. [http://docs.oracle.com/javase/7/docs/technotes/guides/language/strings-switch.html] nicely